REQUIRE

```diff
@@ -1 +1,2 @@
 julia 0.4
+Compat 0.7.14+
```

Comment:
I'm planing to replace the version once a new version is tagged so does leaving many PR's open with this count as "widespread"? The `+` is actually meant to be a reminder to **NOT** merge this PR yet (both by using a valid but unusual version and failing the CI....).

FWIW, this usage came from https://github.com/JuliaLang/JLD.jl/pull/23#discussion-diff-39738508R3 ....